app/src/main/java/com/example/androiddemoproject/recyclerview/chatscreen/ChatScreenActivity.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.androiddemoproject.recyclerview.chatscreen
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView.VERTICAL
+import com.example.androiddemoproject.R
+import com.example.androiddemoproject.databinding.ActivityChatScreenBinding
+import com.example.androiddemoproject.recyclerview.chatscreen.adapter.ChatAdapter
+import com.example.androiddemoproject.recyclerview.chatscreen.model.ChatMessage
+import com.example.androiddemoproject.recyclerview.chatscreen.model.UserType
+import com.example.androiddemoproject.recyclerview.chatscreen.model.getChatMessages
+import com.example.androiddemoproject.recyclerview.chatscreen.model.toChatItems
+import com.example.androiddemoproject.recyclerview.chatscreen.utils.ChatMessageItemDecoration
+import com.example.androiddemoproject.recyclerview.chatscreen.utils.getCurrentDateTime
+
+class ChatScreenActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityChatScreenBinding
+    private val chatAdapter by lazy { ChatAdapter() }
+    private val messages by lazy { getChatMessages().toMutableList() }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityChatScreenBinding.inflate(layoutInflater)
```

Comment:
You can use `DataBindingUtil.setContentView(...)` here which is recommended way to create binding for an Activity.